xstate-chatbot/nodejs/src/machine/hospital-details.js

```diff
@@ -0,0 +1,1068 @@
+            },
+            {
+              cond: (context) => context.isValid == true,
+              target: '#l2HospitalUpdate',
+            },
+          ],
+        },
+        error: {
+          onEntry: assign((context, event) => {
+            dialog.sendMessage(context, dialog.get_message(messages.l2HospitalUpdate.error, context.user.locale), false);
+          }),
+          always: 'prompt',
+        },
+      },
+    },// L2 Hospital comparing time slots
+    l2HospitalUpdate: {
+      id: 'l2HospitalUpdate',
+      initial: 'prompt',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+            context.grammer = grammers.binaryChoice.grammer;
+            let message = dialog.get_message(messages.l2HospitalUpdate.prompt, context.user.locale);
+            message += dialog.get_message(grammers.binaryChoice.prompt, context.user.locale);
+            dialog.sendMessage(context, message);
+          }),
+          on: {
+            USER_MESSAGE: 'process',
+          },
+        },
+        process: {
+          onEntry: assign((context, event) => {
+            context.intention = dialog.get_intention(context.grammer, event, true);
+          }),
+          invoke: {
+            src: (context, event) => bedsService.getHospitalById(context.slots.hospital.mobileNumber),
+            onDone: [
+              {
+                cond: (context, event) => context.intention === 'YES',
+                actions: assign((context, event) => {
+                }),
+                target: '#l2ConfirmedCOVIDPatientsOnOxygen',
+              },
+              {
+                cond: (context, event) => context.intention === 'NO',
+                actions: assign((context, event) => {
+                }),
+                target: '#l2UpdateExistingDetails',
+              },
+              {
+                cond: (context, event) => context.intention === 'INTENTION_UKNOWN',
+                target: 'error',
+              },
+            ],
+          },
+        },
+        error: {
+          onEntry: assign((context, event) => {
+            dialog.sendMessage(context, dialog.get_message(messages.l2HospitalUpdate.error, context.user.locale), false);
+          }),
+          always: 'prompt',
+        },
+      },
+    },// Consent for updating L2 Hospital Details
+    l2UpdateExistingDetails: {
+      id: 'l2UpdateExistingDetails',
+      invoke: {
+        src: (context) => bedsService.updatehospitaldata(context.slots.hospital),
+        onDone: [
+          {
+            actions: assign((context, event) => {
+              let message = dialog.get_message(messages.l2UpdateExistingDetails.prompt, context.user.locale);
+              message += `\n`;
+              message += dialog.get_message(messages.confirmedPreviousCOVIDPatientsOnOxygen.prompt, context.user.locale);
+              message += `\t::\t ${context.slots.previoushospitaldata.confirmed_cases_on_oxygen_support_l2}`;
```

Comment:
\t (Tabs) may or may not look good in a WhatsApp message. You can try it out first from your WhatsApp account.

Reply:
removed from everywhere (wherever used)